### PR TITLE
add notification to user nav bar:WIP

### PR DESCRIPTION
**Pull request by accbjt**
## Description
Add notification to user nav

User Flow:
- log in 
- click on user nav area
- 
## Testing done
local testing

## Acceptance criteria
- [ ] Notification link should be in the user nav area
- [ ] Must be like design mockup [mockup](https:google.com)

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/user-nav/components/PersonalizationDropdown.jsx [hunk 1]
```diff
@@ -47,6 +47,11 @@ class PersonalizationDropdown extends React.Component {
             Account
           </a>
         </li>
+        <li>
+          <a href="/notification" onClick={recordAccountEvent}>
+            Notification
```
Comment (1Copenut) [inline src/platform/site-wide/user-nav/components/PersonalizationDropdown.jsx]: Can we change this to Notifications, plural?

**Review by 1Copenut — CHANGES_REQUESTED** [review]
@accbjt I left a request to pluralize this text.

**Comment by accbjt** [comment]
@1Copenut can you look at this one more time

**Comment by 1Copenut** [comment]
@accbjt Can we close this PR? I think it's our example from the engineering demo.

**Comment by accbjt** [comment]
Yes we can

**Comment by 1Copenut** [comment]
Closing per conversation and demo code only.

**Pull request closed without merging**